docs/integrations/google_colab.rst

```diff
@@ -3,38 +3,108 @@
 Neptune-Google Colab Integration
 ================================
 
-You can run experiments on Google Colab and track them with Neptune.
+..
+    Video [1min screencast]
```

Comment:
Remember to add a short video here -> once you have something uploaded to loom I can work on the Thumbnail and cuts if needed.

Reply:
Video uploaded to Loom team library --> https://www.loom.com/share/a533ef47757a498a9fa944541b56d780